charts/argo-cd/Chart.yaml

```diff
@@ -23,6 +23,8 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
+    - kind: added
```

Comment:
Hi please change this to fixed and removed other non-relevant points below for the new release.

Reply:
Done
Sorry about that, and thanks for the patience.